test/runtests.jl

```diff
@@ -1,10 +1,24 @@
+# path=pwd()
+# if path[end-6:end] != "MatLang"
+# 	error("cd(to the MatLang path)")
+# end
+
+push!(LOAD_PATH, ".")
+using Revise
 using MatLang
 using Test
 
-# usage
-clcM()
-
 # tests
 @testset "MatLang.jl" begin
-1==1
+
```

Comment:
`runtests.jl` is the "main" for tests, all other test files should be included by this file.

```diff
+ include("test/usage.jl")
```

Reply:
I wanted to keep usage.jl as a playground for the developer to test examples and play with the code. If it is not clear I will move it to another folder.

Reply:
If it's only a playground, then you shouldn't upload it.

Reply:
It is meant to be somewhere that user can see, run and play with all the examples I used in the documentation